tests/test_unit/test_tracking_utils.py

```diff
@@ -59,7 +60,7 @@ def test_write_tracked_bbox_to_csv(csv_writer, csv_output):
         1,
         0,
         '"{""name"":""rect"",""x"":10,""y"":20,""width"":40,""height"":60}"',
-        '"{""track"":""1""}"',
+        '"{""track"":""1"", ""pred_score"":""0.9""}"',
```

Comment:
```suggestion
        '"{""track"":""1"", ""confidence"":""0.9""}"',
```
Just because I assumed this for the movement dataloader (see example [here](https://github.com/neuroinformatics-unit/movement/blob/f8052a8151fdba6b5717b9adc695a53cdad013a1/tests/test_unit/test_load_bboxes.py#L58C26-L58C36))